doc/Makefile

```diff
@@ -5,10 +5,10 @@ include Makefile.sphinx
 
 CHPL2RST     = ../util/run-in-venv.bash ./util/chpl2rst.py
 
-CHPL_DOC_PRIMERS_DIR = primers/primers
+CHPL_DOC_PRIMERS_DIR = $(SOURCEDIR)/primers/primers
```

Comment:
I think it could be a separate one, but I think it'd be a nice cleanup either way.